cerbos/sdk/container.py

```diff
@@ -16,5 +16,8 @@ def __init__(self, image: str = "ghcr.io/cerbos/cerbos:latest"):
     def http_host(self, protocol: str = "http") -> str:
         return f"{protocol}://{self.get_container_host_ip()}:{self.get_exposed_port(self.HTTP_PORT)}"
 
+    def grpc_host(self) -> str:
+        return f"{self.get_container_host_ip()}:{self.get_exposed_port(self.HTTP_PORT)}"
```

Comment:
Shouldn't this be `GRPC_PORT`? (It would still work because Cerbos can serve gRPC over the HTTP port as well but there are some limitations to that.)

```suggestion
        return f"{self.get_container_host_ip()}:{self.get_exposed_port(self.GRPC_PORT)}"
```